### PR TITLE
- Change Assembly Name to Akka

### PR DESCRIPTION
**Pull request by haighis**
- Change namespaces to original naming from akka.net
- Move classes back to original folders Configuration and Configuration/Hocon as in akka.net
- Update HelloHocon example to use configuration over getting configuration from string.

Related to #16

### CODE REVIEW

File: src/Hocon.sln [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.31101.0
+# Visual Studio 14
+VisualStudioVersion = 14.0.23107.0
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Hocon", "core\Hocon\Hocon.csproj", "{4A87AD05-BEF2-4B08-BD7E-A9866D88C439}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Akka", "core\Hocon\Akka.csproj", "{4A87AD05-BEF2-4B08-BD7E-A9866D88C439}"
```
Comment (rogeralsing) [inline src/Hocon.sln]: IMO. Akka.Hocon.csproj would be more appropriate, so its not confused with the core Akka project. 
And the same for the assemby name. 

cc @akkadotnet/developers

Comment (haighis) [inline src/Hocon.sln]: This will require a breaking change to all existing instances of

Comment (rogeralsing) [inline src/Hocon.sln]: Ah yes, but we can not have two assemblies with the same name, Akka could not take a dependency on Akka, that would be strange.

Comment (haighis) [inline src/Hocon.sln]: Right that is strange, but I wanted to clarify direction required so we have the impact in the way you want to handle this for existing users.

**Comment by rogeralsing** [comment]
We will bring this up on the next contributor meeting.
I've talked to @Aaronontheweb about it and we have some ideas how to make this not break anything, but we need to chose the right path to go.

**Pull request merged**